lib/core/include/irods/rodsErrorTable.h

```diff
@@ -100,6 +100,7 @@ NEW_ERROR(SYS_UNRECOGNIZED_REMOTE_FLAG,                -20000)
 NEW_ERROR(SYS_INVALID_SERVER_HOST,                     -21000)
 NEW_ERROR(SYS_SVR_TO_SVR_CONNECT_FAILED,               -22000)
 NEW_ERROR(SYS_BAD_FILE_DESCRIPTOR,                     -23000)
+NEW_ERROR(SYS_RESOLUTION_ERROR,                        -23001)
```

Comment:
Append to the end of group.

Make the integer value follow the same pattern as its siblings.